charts/external-dns/values.yaml

```diff
@@ -213,8 +213,59 @@ txtSuffix:
 domainFilters: []
 
 provider:
-  # -- _ExternalDNS_ provider name; for the available providers and how to configure them see the [README](https://github.com/kubernetes-sigs/external-dns#deploying-to-a-cluster).
+  # -- _ExternalDNS_ provider name. When the provider is `webhook` it will add a sidecar using the `webhook` values.
```

Comment:
I don't think the previous comment should be removed here. Also as there are likely to be values other than `webhook` added here listing them here by name won't scale. I'd be happy if we added a section to the README template to link to instead of the main README, this could then link to the main README link and have any chart specific context added.

Reply:
>  I'd be happy if we added a section to the README template to link to instead of the main README, this could then link to the main README link and have any chart specific context added.

I needed to re-read 3 times this sentence in order to understand it :sweat_smile: . 
Anyway, I think I got your point. Wdyt of c5e79337 ?